examples/warehouse/_config/PalletJack2Salt/README

```diff
@@ -0,0 +1,34 @@
+PalletJack2Salt Configuration
+=============================
+
+Type name: _config
+
+Represents warehouse specific configuration for a palletjack tool.
```

Comment:
These two lines don't fit here in the tool-specific documentation. Leave them in the generic `_config` documentation.